README.md

```diff
@@ -211,6 +211,10 @@ Modules or frameworks for virtual machines that run in the Cosmos SDK
 * [REStake](https://restake.app) - Auto-compounder app for Cosmos blockchains using Authz ([source](https://github.com/eco-stake/restake)).
 * [Yieldmos](https://yieldmos.com) - Staking and LP rewards auto-compounder using Authz.
 
+### Faucets
```

Comment:
```suggestion
### Bots
```

Reply:
I thought about creating the `Bots` section at first, but I couldn't decide if it was more of a bot or a faucet, and I went with the faucet.  Tbh I think both are ok, it can go either way. What do you mean by "add the section to the ToC" ?

Reply:
It's just so that we don't end-up with too specific sections.
ToC = Table of Content. Can you please add the `Bots` section there?

Reply:
Sure thing, under tools? or a new section?